res/css/views/messages/_MessageActionBar.scss

```diff
@@ -114,6 +114,14 @@ limitations under the License.
             }
         }
 
+        &.mx_MessageActionBar_favouriteButton::after {
+            mask-image: url('$(res)/img/element-icons/room/message-bar/star.svg');
+        }
+    
+        &.fillStar::after {
+            background-color: #ffa534;
```

Comment:
We need to give this colour a name and make sure it is themeable. For an example, see https://github.com/matrix-org/matrix-react-sdk/blob/develop/res/themes/light/css/_light.scss#L127 `$menu-border-color`